src/components/events/Event.jsx

```diff
@@ -1,5 +1,65 @@
-const Event = () => {
-  return <div>Event</div>;
+"use client";
+import React, { useState } from "react";
+const Event = ({
+  month = "FEB",
+  day = "29",
+  time = "7:00PM",
+  title = "Event Title",
+  link = "http://google.com",
+  description = "Lorem ipsum dolor hello this is a test for the lol",
+  color = "bg-orange-400",
```

Comment:
make sure you're using the colors specified under tailwind.config.js (bg-hlg-orange instead of bg-orange-400)